pyproject.toml

```diff
@@ -27,10 +23,11 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-  "bioio-base>=1.0.0",
-  "fsspec>=2022.8.0",
-  "ome-zarr>=0.8.0",
-  "xarray>=0.16.1",
+    "bioio-base>=1.0.0",
+    "fsspec>=2022.8.0",
+    "ome-zarr>=0.9.0",
+    "xarray>=0.16.1",
+    "zarr>=2.18.2",
```

Comment:
Isnt this a dependency of ome-zarr?

Reply:
Yes I believe so, but now we are using it directly